dfc/config.go

```diff
@@ -344,3 +344,20 @@ func setloglevel(loglevel string) (err error) {
 	}
 	return
 }
+
+// setGLogVMoudle sets glog's vmodule flag
+// sets 'v' as is, no verificaton is done here
+// syntax for v: target=5,proxy=1, p*=3, etc
+func setGLogVMoudle(v string) error {
```

Comment:
`setGLogVMoudle` -> `setGLogVModule`

Reply:
of course
i wonder when will nvidia's AI can do spelling check on source code, but if that day do come, probably we all will lose our jobs
thanks for catching it